src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/UserService.kt

```diff
@@ -355,6 +357,10 @@ class UserService(
     return teams?.firstOrNull()?.borough?.let { probationDeliveryUnitRepository.findByDeliusCode(it.code) }
   }
 
+  private fun orderUserDeliusTeams(deliusUser: StaffUserDetails): List<StaffUserTeamMembership>? {
```

Comment:
It might be slightly easier to read if we rename this to `orderedDeliusUserTeams`

Reply:
function renamed